op-supervisor/supervisor/backend/cross/unsafe_update_test.go

```diff
@@ -114,7 +114,7 @@ func TestCrossUnsafeUpdate(t *testing.T) {
 		// (the first one is in CrossSafeHazards)
 		usd.deps.chainIDFromIndexfn = func() (eth.ChainID, error) {
 			defer func() { count++ }()
-			if count == 1 {
+			if count < 2 {
```

Comment:
the comment says "by failing the second ChainIDFromIndex call", but this appears to only fail starting at the 3rd call?